packages/sanity/src/core/field/diff/components/DiffTooltip.tsx

```diff
@@ -56,13 +56,7 @@ function DiffTooltipWithAnnotation(props: DiffTooltipWithAnnotationsProps) {
 
   return (
     <LegacyLayerProvider zOffset="paneFooter">
-      <Tooltip
-        content={content}
-        // data-placement={restProps.placement}
-        portal
-        allowedAutoPlacements={['top', 'bottom']}
```

Comment:
Removed as it's now deprecated in Sanity UI, and including it would yield similar transient prop warnings